SQL/paradise_schema.sql

```diff
@@ -79,6 +79,7 @@ CREATE TABLE `characters` (
   `hair_gradient_colour` varchar(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '#000000',
   `hair_gradient_alpha` tinyint(3) UNSIGNED NOT NULL DEFAULT '255',
   `custom_emotes` LONGTEXT COLLATE 'utf8mb4_unicode_ci' DEFAULT NULL,
+  `cyborg_brain_type` varchar(11) NOT NULL DEFAULT 'MMI',
```

Comment:
Make this an enum in the DB for the 3 types, saves on disk space and parsing time.